kolibri/plugins/coach/assets/src/views/plan/LessonSummaryPage/ResourceListTable.vue

```diff
@@ -28,32 +28,42 @@
               </div>
             </KFixedGridItem>
             <KFixedGridItem span="4">
-              <div class="resource-title">
-                <div class="content-icon">
-                  <ContentIcon :kind="resourceKind(resourceId)" />
+              <template v-if="getCachedResource(resource.contentnode_id)">
+                <div class="resource-title">
+                  <div class="content-icon">
+                    <ContentIcon :kind="resourceKind(resource.contentnode_id)" />
+                  </div>
+                  <div class="content-link">
+                    <KRouterLink
+                      :text="resourceTitle(resource.contentnode_id)"
+                      :to="$router.getRoute(
+                        'RESOURCE_CONTENT_PREVIEW', { contentId: resource.contentnode_id }
+                      )"
+                    />
+                  </div>
+                  <p dir="auto" class="channel-title" :style="{ color: $themeTokens.annotation }">
+                    <dfn class="visuallyhidden"> {{ $tr('parentChannelLabel') }} </dfn>
+                    {{ resourceChannelTitle(resource.contentnode_id) }}
+                  </p>
                 </div>
-                <div class="content-link">
-                  <KRouterLink
-                    :text="resourceTitle(resourceId)"
-                    :to="$router.getRoute('RESOURCE_CONTENT_PREVIEW', { contentId: resourceId })"
-                  />
-                </div>
-                <p dir="auto" class="channel-title" :style="{ color: $themeTokens.annotation }">
-                  <dfn class="visuallyhidden"> {{ $tr('parentChannelLabel') }} </dfn>
-                  {{ resourceChannelTitle(resourceId) }}
+                <CoachContentLabel
+                  class="coach-content-label"
+                  :value="getCachedResource(resource.contentnode_id).num_coach_contents"
+                  :isTopic="false"
+                />
+              </template>
+              <template v-else>
+                <p>
+                  <KIcon icon="warning" :style=" { fill: $themePalette.orange.v_400 }" />
+                  {{ resourceMissingText }}
```

Comment:
can these be implemented with `KLabeledIcon`?

Reply:
Also there is a `color` prop on `KIcon`

Reply:
Using KLabeledIcon makes the appearance and formatting significantly worse:

![Screenshot from 2020-08-04 11-48-37](https://user-images.githubusercontent.com/1680573/89332574-76f55380-d648-11ea-9347-7f3ff7ac3740.png)

vs with just a <p>:
![Screenshot from 2020-08-04 11-49-24](https://user-images.githubusercontent.com/1680573/89332650-99876c80-d648-11ea-97c0-1ff06d524255.png)

Reply:
Is the bad one a KLabeledIcon in a <p> or by itself?

Reply:
The first one is with the KLabeledIcon and I think it gives a much worse result than the second one which is using a `<p>` tag

Reply:
Oh, the KLabeledIcon is by itself, sorry, no p

Reply:
Fine to leave for now, but FYI that looks like something was misconfigured. From `KLabeledIcon`:

```scss
  .debug-warning > svg {
    // if you see this, you need to pass the icon into the slot
    border: 1px solid red;
  }
```